eng/Versions.props

```diff
@@ -6,7 +6,7 @@
     <PatchVersion>0</PatchVersion>
     <PreReleaseVersionLabel>rtm</PreReleaseVersionLabel>
     <!-- Enable to remove prerelease label. -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
```

Comment:
Destabilizing was already done in https://github.com/dotnet/core-setup/pull/8889 for this branch, but it's fine: a local merge works fine and doesn't include this diff.